### DISCUSSION TITLE
Adding a website to the desktop tool

### DISCUSSION THREAD

**Discussion by Aegolius76** [chilli-axe/mpc-autofill#268] [Q&A]
Hi,

I'm new to Github and I need some help to adapt the code to add a new website choice to the desktop tool.

I'm in France and we have the website PrinterStudio.fr. It's the same as PrinterStudio.de which is already supported. 
I'm trying to add the french site to the autofill.exe 

I've downloaded the sourcefiles and found 2 files in the desktop-tool directory with the words PrinterStudio

I've added some code in those 2 files desktop-tool\src\constants.py & desktop-tool\tests\test_desktop_client.py matching the lines for PrinterstudioDE

I don't know how to compile the new code to an .exe file and test if it works on the french website.

I've made 2 pull requests with the new code

Thanks,

Antonin

**Comment by Michael833** [comment]
Similarly, www.notmpc.com seems to use basically the same interface, and is cheaper and faster for shipping to the US. Is that a site that could be supported, or are there too many differences to the other sites?

**Reply by ndepaola** [reply]
Regarding notmpc - see this thread https://github.com/chilli-axe/mpc-autofill/discussions/351
